zilencer/management/commands/populate_db.py

```diff
@@ -332,6 +332,9 @@ def handle(self, *args: Any, **options: Any) -> None:
 
             realm_user_default = RealmUserDefault.objects.get(realm=zulip_realm)
             realm_user_default.enter_sends = True
+            realm_user_default.email_address_visibility = (
+                RealmUserDefault.EMAIL_ADDRESS_VISIBILITY_ADMINS
+            )
```

Comment:
Why does this need parens -- is it just for better line-wrapping?

Reply:
Parens got added by black.